### PR TITLE
feat(fast-components-react-msft): adding density input to development site

### PR DESCRIPTION
**Pull request by khamudom**
- density slider is added and hooked up to work #907

### CODE REVIEW

File: packages/fast-components-react-msft/app/app.tsx [hunk 1]
```diff
@@ -165,6 +180,13 @@ export default class App extends React.Component<{}, IAppState> {
         );
     }
 
+    private handleDensityUpdate = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        const value: number = +e.target.value;
```
Comment (nicholasrice) [inline packages/fast-components-react-msft/app/app.tsx]: I think you'll want to change this to `const value: number = parseInt(e.target.value, 10);` - `e.target.value` is a string and this is required to be a number.

**Comment by awentzel** [comment]
Can you update the title to conform to standards defined [here](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance)

File: packages/fast-components-react-msft/app/app.tsx (hunk 1 above)
Comment (khamudom) [reply packages/fast-components-react-msft/app/app.tsx]: made the changed

File: packages/fast-components-react-msft/app/app.tsx [hunk 2]
```diff
@@ -100,12 +102,24 @@ export default class App extends React.Component<{}, IAppState> {
...
                 </SiteCategory>
                 <div slot={ShellSlot.infoBar}>
-                    <ColorPicker
-                        foregroundColor={this.state.foregroundColor}
-                        backgroundColor={this.state.backgroundColor}
-                        accentColor={this.state.accentColor}
-                        onColorUpdate={this.handleColorUpdate}
-                    />
+                    <span style={{display: "flex", alignItems: "center", height: "100%"}}>
+                        <label style={{marginRight: "8px"}}>density</label>
```
Comment (chrisdholt) [inline packages/fast-components-react-msft/app/app.tsx]: do we want this to be lowercase "d"? Also, any reason we aren't using the Label component here?

Comment (khamudom) [reply packages/fast-components-react-msft/app/app.tsx]: switched to div and Label

Comment (khamudom) [reply packages/fast-components-react-msft/app/app.tsx]: I used a lowercase "d" because the theme colors are lowercase.

**Comment by khamudom** [comment]
Abandoning this pull request. Creating a new one for this change.

**Pull request closed without merging**